res/schemas/login.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "email": {
+      "type": "string",
+      "format": "email"
+    },
+    "password": {
+      "type": "string"
```

Comment:
Can we add minimum length constraint here as well? Since we remove the if condition in code.

Reply:
I think it's generally not a good idea to check constraints on login because if they change then users may not be able to access their account.